packages/zowe-explorer-ftp-extension/resources/conformance/test_evaluation_guide_table.md

```diff
@@ -0,0 +1,319 @@
+# Zowe Conformance Test Evaluation Guide
```

Comment:
Should this file be moved to the top-level `docs` folder ?
I believe some links may not be working right now 😢